modules/gdscript/gdscript_parser.h

```diff
@@ -1323,6 +1323,7 @@ class GDScriptParser {
 		int current_line = -1;
 		int current_argument = -1;
 		Variant::Type builtin_type = Variant::VARIANT_MAX;
+		bool has_param_parenthesis = false;
```

Comment:
Those things could probably be an union, but are those few byte savings are really worth the potential problems associated to it? I say nah.